nss-day-cohort-51/nss-day-cohort-51.github.io
# Description Section: Rogue white line showing on right of SVG design mw-fix-Description-section-147 Redrew the background SVG for the Desc…

Redrew the background SVG for the Description section above the fold; fixed the ghost line caused from the origin svg not being flush to the right side of it's invisible box/frame.

# Description

## Type of change


# Testing Instructions
- pull down branch and serve website using npm start
- in the browser, mouseover the right side of the Description section/above the fold and see that there is no longer a thin white gap to the right of the section


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
